frontstage/common/redis_cache.py

```diff
@@ -18,7 +21,8 @@
 class RedisCache:
     SURVEY_CATEGORY_EXPIRY = 600  # 10 mins
     COLLECTION_INSTRUMENT_CATEGORY_EXPIRY = 600  # 10 mins
-    BUSINES_PARTY_CATEGORY_EXPIRY = 600
+    BUSINES_PARTY_CATEGORY_EXPIRY = 600  # 10 mins
```

Comment:
Can you change the name of this to be the correct spelling of business?  I missed it in the last PR when it was added